.circleci/config.yml

```diff
@@ -62,7 +68,6 @@ jobs:
       #    destination: scan-logs
 
 workflows:
-  version: 2
-  build-and-deploy:
-    jobs:
-      - build-and-test
+    build_and_tests:
```

Comment:
Bit of a nitpick - and I could be wrong - but should this be indented 2 spaces instead of 4?